src/ucp/core/ucp_context.c

```diff
@@ -89,9 +89,21 @@ static ucs_config_field_t ucp_config_table[] = {
    " - dc/dc_x : dc with accelerated verbs.\n"
    " - tcp     : sockets over TCP/IP.\n"
    " Using a \\ prefix before a transport name treats it as an explicit transport name\n"
-   " and disables aliasing.\n",
+   " and disables aliasing.",
    ucs_offsetof(ucp_config_t, tls), UCS_CONFIG_TYPE_STRING_ARRAY},
 
+  {"MEMTYPE_TLS", UCP_RSC_CONFIG_ALL,
+   "Comma-separated list of non host memory type transports to use.\n"
+   "The order is not meaningful.\n"
+   " - all       : use all the available transports.\n"
+   " - cuda_ipc  : CUDA IPC API.\n"
```

Comment:
if I'm not mistaken cuda_ipc and rocm_ipc can't be used as memtype TLs (i.e. to detect memory type and transfer from device to host and vice versa)

Reply:
yes. we have to have a basic mem type TL ( i.e cuda_copy, rocm_cpy) for mem type support. things may not work if we specify cuda_ipc w/o cuda_copy.

I started a PR: https://github.com/openucx/ucx/pull/3999 to prevent this usage. I didn't get a chance to finish this.

Reply:
yes, it aligns the current behavior - if you set just `UCX_TLS=cuda_ipc` - it will not work. However `cuda_ipc` can still be considered as a memtype transport. It is a bit similar to enabling just `rc` without `ud`, which is possible to do, but it will not work. 

To fix that, I'd uses aliases, like we do with regular transports, so setting `UCX_MEMTYPE_TLS` to cuda_ipc will result in `cuda_ipc,cuda_copy`.